business-materials/00_事業紹介サマリー.md

```
@@ -1,137 +1,151 @@
-| 2枚目以降 | 500円/枚 |
-| 100枚以上 | 300円/枚 |
-
-### 高性能ホームページ制作
-| プラン | 月額料金 |
-|--------|---------|
-| ビギナー | 15,000円 |
-| スタンダード | 25,000円 |
-| プレミアム | 30,000円 |
+### TapForge（NFCデジタル名刺）
+
+| 項目                | 価格                |
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)